libs/shared/ui/src/lib/styles/main.scss

```diff
@@ -29,6 +29,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   background: theme('colors.element.light.lighter.400');
+  min-height: 100vh;
```

Comment:
To fix what did you needed that?

Reply:
Because on the layout to create a flow, we have an height bug when we displaying a modal 
You can look here: https://www.loom.com/share/a42b008038bc45af9936d5c1d747b68b